scripts/fetch/fetch.islands.R

```diff
@@ -0,0 +1,38 @@
+#' take map arguments and return a projected sp object
+#' 
+#' @param \dots arguments passed to \code{\link[maps]{map}} excluding \code{fill} and \code{plot}
+#' 
+library(maps)
+library(mapdata)
+library(sp)
+to_sp <- function(..., proj.string, within = NULL){
```

Comment:
is `to_sp` going to get used everywhere? maybe extract it into a helper already?

Reply:
where do we put helper functions?

Reply:
helper functions can be put wherever, but they need to be included in the `scripts` for each thing using them

Reply:
just another script that's listed in scripts. probably scripts/fetch for this one, so could be scripts/fetch/to_sp.R